pkg/scheduler/framework/score.go

```diff
@@ -17,14 +17,14 @@ type ClusterScore struct {
 	// AffinityScore determines how much a binding would satisfy the affinity terms
 	// specified by the user.
 	AffinityScore int
-	// BoundOrScheduledScore reflects if there has already been an active/creating binding from
+	// PrevBoundOrScheduledScore reflects if there has already been an obsolete binding from
 	// the same cluster resource placement associated with the cluster; it value range should
-	// be [0, 1], where 1 signals that an active/creating binding is present.
+	// be [0, 1], where 1 signals that an obsolete binding is present.
 	//
 	// Note that this score is for internal usage only; it serves the purpose of implementing
 	// a preference for already selected clusters when all the other conditions are the same,
 	// so as to minimize interruption between different scheduling runs.
-	BoundOrScheduledScore int
+	PrevBoundOrScheduledScore int
```

Comment:
```suggestion
	ObsoletePlacementAffinityScore int
```

how about this name? it will be populated by the SamePlacementAffinity plugin

Reply:
Done.